server/models/video/video-channel.ts

```diff
@@ -287,6 +287,12 @@ export type SummaryOptions = {
               ')'
             ),
             'viewsPerDay'
+          ],
+          [
+            literal(
+              '(SELECT COALESCE(SUM("video".views), 0) AS totalViews FROM "video" WHERE "video"."channelId" = "VideoChannelModel"."id" GROUP BY "video"."channelId")'
```

Comment:
I have a lot of trouble with that query.
All I get while running it is:
```
Backend returned code 500, errorMessage is: syntax error at or near "SELECT"
```

And if I run it inside psql, here's what it gives me:
```
ERROR:  missing FROM-clause entry for table "VideoChannelModel"
LINE 1: ...talViews FROM "video" WHERE "video"."channelId" = "VideoChan...
                                                             ^
```

Reply:
Backend seems to work well on my side

Reply:
You mean, the query's working?

Reply:
Yes and the controllers correctly set the `totalViews` attribute for channels

Reply:
Okay. That may be caused by Gitpod then. I'll investigate that further, some time in the week.